game/src/Credits/Credits.gd

```diff
@@ -0,0 +1,98 @@
+extends Control
+
+signal backButtonPressed
+
+###############
+# Credits CSV format
+# The project title row is the only requirement within the csv file, however
+# it can be on any row, so long as it exists.
+# ----------------------
+# title,project-title
+# role-name,person-name
+# role-name,person-name
+# role-name,person-name
+# ...
+###############
+
+@export var coreCreditsPath : String = 'res://common/credits.csv'
+
+# REQUIREMENTS:
+# * 1.5 Credits Menu
+# * SS-17
+
+# REQUIREMENTS
+# * FS-4
+func loadCreditFile(path):
+	var roles = {}
+	var coreCredits = FileAccess.open(path, FileAccess.READ)
+	
+	while not coreCredits.eof_reached():
+		var line = coreCredits.get_csv_line()
+		
+		# If the line does not have an identifiable role or is empty then skip it
+		if line[0] == '':
+			continue
+		
+		var role: String = line[0].to_upper()
+		var person: String = line[1]
+		
+		if role not in roles.keys():
+			roles[role] = []
+			roles[role].push_back(person)
+		else:
+			roles[role].push_back(person)
+			
+	return roles
+
+# REQUIREMENTS:
+# * UI-34, UI-35
+func drawProjectCredits(project):
+	var projectParent = VBoxContainer.new()
+	var projectLabel = Label.new()
+	
+	projectParent.name = 'Credits-' + project['TITLE'][0]
+	projectLabel.name = 'Label-' + project['TITLE'][0]
+	projectLabel.text = project['TITLE'][0]
+	projectLabel.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
+	projectLabel.label_settings = load("res://src/Credits/ProjectLabelSettings.tres")
+	
+	for role in project.keys():
+		if role == 'TITLE':
+			continue
+		
+		var roleParent = VBoxContainer.new()
+		var roleLabel = Label.new()
+		
+		roleParent.name = 'Role-' + role
+		roleLabel.name = 'Label-' + role
+		roleLabel.text = role
+		roleLabel.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
+		roleLabel.label_settings = load("res://src/Credits/RoleLabelSettings.tres")
+		
+		for person in project[role]:
+			var personLabel = Label.new()
+			
+			personLabel.name = 'Label-' + person
+			personLabel.text = person
+			personLabel.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
+			personLabel.label_settings = load("res://src/Credits/PersonLabelSettings.tres")
+			roleParent.add_child(personLabel)
+		
+		projectParent.add_child(roleLabel)
+		projectParent.add_child(roleParent)
+		projectParent.add_child(HSeparator.new())
+	
+	$"Scroll/Projects".add_child(projectLabel)
+	$"Scroll/Projects".add_child(projectParent)
```

Comment:
Avoid relying on hardcoded scene paths, instead you should have a
```gdscript
@export
var projects_node : Control
```
that you assign in the scene editor so that you can refactor the scene in the scene editor without issue and one can reassign the scene if it gets deleted or lost somehow. Avoid hardcoded paths and strings.

Reply:
Will do, it was already giving me issues when I was fiddling with the display so I'll clean that up as well